src/api/supabase/queries/user_queries.ts

```diff
@@ -16,87 +16,166 @@ const supabaseApiKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY;
 // Initialize the Supabase client
 const supabase = createClient(supabaseUrl ?? '', supabaseApiKey ?? '');
 
-export async function fetchUserData(): Promise<
-  PostgrestSingleResponse<User[]> | { data: never[]; error: PostgrestError }
-> {
-  try {
-    const { data: users, error } = await supabase
-      .from('profiles')
-      .select('*')
-      .single();
-
-    if (error) {
-      console.error('Error fetching data:', error);
-      return { data: [], error };
-    }
+export async function fetchUserData() {
```

Comment:
is there a use case for fetching all users? if not you can probably delete this function. if so, nit but i'd suggest renaming to fetchAllUsers